marketplace/oca-repository/values.yaml

```diff
@@ -0,0 +1,7 @@
+ingress:
+  host: oca-repository.marketplace.nextgen.hiro-develop.nl
```

Comment:
please update your ingress to be like the other services 
ingress:
  host: nextgen.hiro-develop.nl
  
then you can put the others part inside of the domain in your helm chart and you should depend on the namespace that will be passed from here instead of making it static 

you can use this as a reference for you 
https://github.com/HIRO-MicroDataCenters-BV/ds-catalog/blob/update/daemonset-to-deployment/server/charts/server/templates/ingress.yaml